src/lightning_app/cli/cmd_pl_init.py

```diff
@@ -148,8 +149,17 @@ def print_pretty_report(
         text_pathname.stylize(f"link file://{path}")
         text_pathname.append(f" {padding} {help_text}", "blue")
 
-        icon = "📂 " if path.is_dir() else "📄 "
-        tree.add(Text(icon) + text_pathname)
+        icon = ""
+        try:
+            if path.is_dir():
+                icon = "📂 ".encode()
```

Comment:
```suggestion
                icon = "📂 ".encode(sys.stdout.encoding)
```
This is needed, otherwise it won't fix the issue. 

The issue occurs when for example Python is encoding the string in utf8 but the terminal is set to display ascii.